.github/workflows/cicd-main.yml

```diff
@@ -4415,687 +3255,533 @@ jobs:
     #      rm -rf examples/nlp/language_modeling/gpt_pretrain_results"
     #      rm -rf examples/nlp/language_modeling/gpt_index_mappings"
     #    }
-    #  }
```

Comment:
add line back since rest is left

Reply:
i don't undertstand

Reply:
just some comments. Since we're not removing the whole comments (lines above), put this line back also